### ISSUE TITLE
setup-boostlog crash bot

### ISSUE THREAD

**Issue by DekuItIs** [Tomato6966/Multipurpose-discord-bot#36]
## **What's your nodejs Version.**
> Show the output of: `node -v`
16.30.0

![Screenshot 2022-01-29 210508](https://user-images.githubusercontent.com/48065207/151686193-aa429d52-550a-48ef-936e-d765c47b645c.png)

## **What's the version you downloaded the bot?**
> From the releases e.g.: latest

## **Describe the bug**
Everytime you do setup-boostlog and complete one of the interactions you get the error message and the bot crashes.

## **To Reproduce**
> Steps to reproduce the behavior e.g:
1. Execute the Command 'setup-boostlog'
2. Click an interaction
3. Complete the interaction
2. See error

## **Expected behavior**
> being able to setup the boostlog

## **Additional context, text and screenshots!**
> Add any other context about the problem here.

**Comment by Tomato6966** [comment]
Will be checked soon

**Comment by jedd0888** [comment]
> Will be checked soon

![image](https://user-images.githubusercontent.com/38686545/152674804-1caa6858-bd70-4bb0-a104-8f0e1be93682.png)


i found the bug?
you are setting the collection, dont you want to set message.content?

**Comment by jedd0888** [comment]
well when i did that at least it fixed it for me

**Comment by jedd0888** [comment]
https://github.com/Tomato6966/Multipurpose-discord-bot/pull/56/commits/aa5dd75198888f60cf7e28ed11f7d7e92c621806

**Comment by jedddg** [comment]
@Tomato6966 ^^ check this, the bug was reported on discord too and this is the fix

**Comment by mooogah** [comment]
😜

**Comment by M3rcena** [comment]
> 😜 this isn't my issue

Yeah just noticed! I am sorry. I am still waiting for the permissions so I can start closing issues
